packages/wdio-browserstack-service/src/service.ts

```diff
@@ -66,6 +66,7 @@ export default class BrowserstackService implements Services.ServiceInstance {
         if (process.env.BROWSERSTACK_TURBOSCALE) {
             this._turboScale = process.env.BROWSERSTACK_TURBOSCALE === 'true'
         }
+        process.env.BROWSERSTACK_TURBOSCALE_INTERNAL = this._turboScale + ''
```

Comment:
Why the `+ ''` can we properly type `_turboScale`?

Reply:
_turboScale is meant to be of type boolean so to have the environment variable use this, I've added an explicit cast to data type string.